kpops/config/__init__.py

```diff
@@ -162,3 +166,14 @@ def settings_customise_sources(
             dotenv_settings,
             file_secret_settings,
         )
+
+
+def get_config() -> KpopsConfig:
+    if not KpopsConfig._instance:
+        msg = f"{KpopsConfig.__name__} has not been initialized, call {KpopsConfig.__name__}.{KpopsConfig.create.__name__}"
+        raise RuntimeError(msg)
```

Comment:
Instead of raising an error, wouldn't it be more convenient to initialize it?

Reply:
then we would need to pass the arguments for `KpopsConfig.create()`

Reply:
Yeah that is true. I like it more like this.